src/components/CreateEmail.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```

Comment:
Lets put this file under src/utils/createEmails.js 

For non components we can stick to lowercase first character